Tema-6/2/style.css

```diff
@@ -0,0 +1,68 @@
+body {
+    max-width: 1200px;
+    max-height: 400px;
+    font-family: 'Times New Roman';
+    font-size: 16px;
+    margin: 10px auto;
+  
+}
+
+section {
+    display: flex;
```

Comment:
Poti sa lasi doar flex fara flex-direction cand vrei sa asezi niste elemente in row pentru ca default flex aseaza elementele in row. Doar daca le vrei sub forma de column atunci se specifica acel flex-direction: column.